src/util.c

```diff
@@ -359,6 +359,18 @@ const char *get_environment(const char *key)
     return getenv(key);
 }
 
+#ifdef _WIN32
```

Comment:
I'll have to work more on this one. I found the code below on an ancient mailing list thread about this problem, so I'm sure it's not the best.